include/rdma/fabric.h

```diff
@@ -297,6 +297,7 @@ struct fi_info {
 	uint64_t		caps;
 	uint64_t		mode;
 	uint32_t		addr_format;
+	uint32_t		version;
```

Comment:
We may want to consider requiring that apps use fi_allocinfo/fi_dupinfo, and no longer support apps that allocate fi_info themselves.